benchmarks/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@ project(xlnt.benchmarks)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Require C11 compiler
+set(CMAKE_C_STANDARD 11)
```

Comment:
I don't mind that much as it is not the main project, but do you have any reason why you increased the required C version from C99 to C11?

Reply:
This should only make a difference for C code (see my other comment). The reason why I changed it was because I thought we wouldn't support C99 anymore, but it makes sense to support it, as it's the default for C++11. By the way, I didn't increase it from C99 to C11, but set it from undefined (whatever the compiler-default setting is) to C11. However, effectively, the benchmarks don't contain any `.c` files, so the code should always be compiled as C++, meaning that this setting makes no difference at the moment. I can also revert it if you think it might be more confusing than it helps :slightly_smiling_face:

Reply:
I would prefer removing or setting to c99.

Reply:
I removed it, as it didn't make any difference here, so probably it would confuse more than it helps.